utils/spacewalk-common-channels.ini

```diff
@@ -205,19 +205,19 @@ repo_url = http://mirrorlist.centos.org/?release=8&arch=%(arch)s&repo=PowerTools
 name     = Uyuni Client Tools for %(base_channel_name)s
 archs    = %(_x86_archs)s, ppc64le, aarch64
 base_channels = centos8-%(arch)s
-gpgkey_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/CentOS8-Uyuni-Client-Tools/CentOS_8/repodata/repomd.xml.key
+gpgkey_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/EL8-Uyuni-Client-Tools/EL_8/repodata/repomd.xml.key
```

Comment:
I know this sounds strange, but yes. This is to keep coherence with the SQL script.

Remember: `spacewalk-common-channels` reuse existing repositories if they share the URL. Meaning if you added CentOS8 first and then you add Oracle8, the first URL will be used and there will be a single repo used for two channels.

So the SQL script will update the URL, regardless of the repo label for this reason, and even CentOS8 will start the EL8 client tools. Which should not be a problem anyway, as you could consider that now CentOS8 is a clone of AlmaLinux 8 (which is what we use for the EL8 client tools)